app/script/event/EventRepository.coffee

```diff
@@ -355,7 +355,7 @@ class z.event.EventRepository
         return @cryptography_repository.decrypt_event event
         .catch (decrypt_error) =>
           # Get error information
-          error_code = decrypt_error.code
+          error_code = decrypt_error.code or 999
```

Comment:
`666` is the [number of the beast](https://en.wikipedia.org/wiki/Number_of_the_Beast) and I don't want to have Satan in our app. 😋